File: Basic_Problem_solving/Day_1.js
//01. Check whether a number is even or odd
//use variable
var num=41;
if(num % 2 == 0)
    console.log("is even.", num);
else
console.log("%d is odd.", num);

//use user input 
let num = prompt("Please enter your name:");
if (num % 2 == 0) {
    console.log("is even.", num);
}
else {
    console.log("%d is odd.", num);
}

//02. Check whether a character is a vowel or consonant
var c = 'a';
var lowercase_vowel = (c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u');;
var uppercase_vowel = (c == 'A' || c == 'E' || c == 'I' || c == 'O' || c == 'U');

if (lowercase_vowel || uppercase_vowel) {
    console.log("Is a vawel");
}
else {
    console.log("consonent");
}
var c = prompt("");
var lowercase_vowel = (c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u');;
var uppercase_vowel = (c == 'A' || c == 'E' || c == 'I' || c == 'O' || c == 'U');

if (lowercase_vowel || uppercase_vowel) {
    console.log("Is a vawel");
}
else {
    console.log("consonent");
}

//switch case
let letter = prompt("Enter a letter:");

switch(letter.toLowerCase()){
    case 'a':
    case 'i':
    case 'o':
    case 'u':
    console.log(letter + " is a vowel.");
    
    break;
    default:
        console.log("Consonent yors inter value")
        
    
}




//03. Find the largest number among three numbers
//normal variable 
var a = 267567; var b = 249; var c = 29; 
//functionality i mean function 
if (a > b && a > c) {
    console.log("big number is ", a);
}
else if (b > a && b > c) {
    console.log("big number is ", b);
}
else {
    console.log("big number is ", c);
}


// user input
var a = prompt("Enter number a:")
var b = prompt("Enter number b:")
var c = prompt("Enter number c:")

if (a > b && a > c) {
    console.log("big number is ", a);
}
else if (b > a && b > c) {
    console.log("big number is ", b);
}
else {
    console.log("big number is ", c);
}

//switch_case
let num1 = parseInt(prompt("Enter the first number:"));
let num2 = parseInt(prompt("Enter the second number:"));
let num3 = parseInt(prompt("Enter the third number:"));

let largest;

switch (true) {
    case (num1 > num2 && num1 > num3):
        largest = num1;
        break;
    case (num2 > num1 && num2 > num3):
        largest = num2;
        break;
    case (num3 > num1 && num3 > num2):
        largest = num3;
        break;
    default:

}

console.log("The largest number is " + largest);

File: Basic_Problem_solving/Day_1.c
// Check whether a number is even or odd => variable && input user
int num=50;
    if(num%2 == 0){
        printf("%d your number is event",num);
        
    }
    else printf("Number is odd");

    // get user input 
    int num;
    printf("Please provide your number: ");
    scanf("%d", &num);
    if (num % 2 == 0)
    {
        printf("%d your number is event", num);
    }
    else
        printf("Number is odd");

   // Check whether a character is a vowel or consonant = > variable &&input user &&switch_case char c;
    printf("Please provide your charcter: ");
    scanf("%c", &c);

    int lowercase_vowel, uppercase_vowel;
    lowercase_vowel = (c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u');
    uppercase_vowel = (c == 'A' || c == 'E' || c == 'I' || c == 'O' || c == 'U');
    if (lowercase_vowel || uppercase_vowel)
    {
        printf("%c Number is vowel", c);
    }
    else
    {
        printf("%c number is consonent", c);
    }


    //switch case
    char c;
    printf("Please provide your charcter: ");
    scanf("%c", &c);

    switch (c)
    {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
    case 'A':
    case 'E':
    case 'I':
    case 'O':
    case 'U':
        printf("%c vowel is a ", c);
        break;

    default:
        printf("%c consonent is a ", c);
    }



// Find the largest number among three numbers
    int number1;
    int number2;
    int number3;
    
    printf("please provie 1 number:");
    scanf("%d",&number1);
    printf("please provie 2 number:");
    scanf("%d",&number2);
    printf("please provie 3 number:");
    scanf("%d",&number3);
    printf("\nyour input  number : %d",number1);
    printf("\nyour input  number : %d",number2);
    printf("\nyour input  number : %d",number3);
    printf("\n");
    
    if(number1>number2 && number1>number3){
        printf("Big number is %d ",number1);
    }
    else if(number2>number1 && number2>number3){
       printf("Big number is %d ",number2);  
    }
    else{
       printf("Big number is %d ",number3);  
    }

#include <stdio.h>

    int main()
    {
       int num1, num2, num3, largest;

       printf("Enter three numbers: ");
       scanf("%d %d %d", &num1, &num2, &num3);

       switch (1)
       {
       case 1:
           if (num1 > num2 && num1 > num3)
           {
               largest = num1;
           }
           else if (num2 > num3)
           {
               largest = num2;
           }
           else
           {
               largest = num3;
           }
           break;
       default:
           printf("Invalid choice\n");
           break;
       }

       printf("The largest number is %d\n", largest);

       return 0;
    }
